src/rgw/rgw_sal_motr.cc

```diff
@@ -786,11 +775,15 @@ bool MotrObject::is_expired() {
   return false;
 }
 
+// Taken from rgw_rados.cc
 void MotrObject::gen_rand_obj_instance_name()
 {
-   //store->getDB()->gen_rand_obj_instance_name(&key);
-}
+#define OBJ_INSTANCE_LEN 32
```

Comment:
I know there is rgw code that does this preprocessor thing inline, but I don't think it's great;  on the one hand, we can use constexpr--but more important, this seems like a definition that shouldn't be at local scope?

Reply:
To be honest, we just copy-pasted it from dbstore:
https://github.com/ceph/ceph/blob/d70fdd74b993610fb0ff26c0a96e040ffa3ddaba/src/rgw/store/dbstore/common/dbstore.cc#L965-L968

Reply:
yes.. Even DBStore is still wip and not fully complete. Need to audit and cleanup the code such as this if not used anywhere. For now focusing on getting the major s3 functionality work done. 

Now that there is another Motr SAL now, we probably can combine this effort.

Reply:
Changed to enum constant.